pkg/controllers/scheduling/scheduling_controller.go

```diff
@@ -452,7 +452,9 @@ func (c *schedulingController) bind(
 	// maxNumOfClusterDecisions.
 	decisionSlices := [][]clusterapiv1beta1.ClusterDecision{}
 	remainingDecisions := clusterDecisions
-	for index := 0; len(remainingDecisions) > 0; index++ {
+	// if clusterDecisions is empty, append empty slice to decisionSlices,
+	// and create an PlacementDecision with empty decisions in status.
+	for index := 0; len(remainingDecisions) > 0 || len(decisionSlices) == 0; index++ {
```

Comment:
This is confusing, can we make it more explicit

Reply:
fixed.